.github/workflows/cron.yml

```diff
@@ -0,0 +1,30 @@
+name: cron
+
+on:
+  schedule:
+  # Run every day at 00:00 UTC
+  - cron: 0 0 * * *
```

Comment:
Why not add `push` too?

Reply:
Maybe a later. The reason is in https://github.com/jazzband/pip-tools/pull/875#issuecomment-537260079.

Reply:
And I think push hook should be in a separate workflow file.

Reply:
Ok, can be done in a separate PR, but I would not do it in a separate file unless you want to execute a different suite.

Reply:
If we find a way how to make "allow_failures" for "pipmaster" except for a cron job, then we can do it in a single workflow suite.